docs/src/main/asciidoc/feature-management.adoc

```diff
@@ -62,7 +62,16 @@ feature-management:
 * `feature-t` is set to false, this setting will always return the Feature Flag's value. 
 * `feature-u` is used in conjunction with Feature Filters. These Filters will be defined under the `enabled-for` property.  In this case, `feature-u` has one Feature Filter called `Random`, which does not require any configuration, so only the name property is required. 
 * `feature-v` specifies a Feature Filter named `TimeWindowFilter`. This Feature Filter can be passed parameters to use as configuration. In this case, for a `TimeWindowFilter`, pass in the start and end times for which the feature will be active.
-* `feature-w` is used for the `AlwaysOnFilter`, which will always evaluate to `true`. The `evaluate` field is used to stop the evaluation of the Feature Filters, and results in the Feature Filter always return `false`.
+* `feature-w` is used for the `AlwaysOnFilter`, which will always evaluate to `true`. The `evaluate` field is used to stop the evaluation of the Feature Filters, and results in the Feature Filter always return `false`.
+
+==== Feature Variants
+
+Feature Variants are multiple options of a feature that can be targeted at different groups of users. A Feature Variant is defined by four values:
+
+* name - Identifier for the varieant.
+* default - Indicates which variant should be used if no variant is assigned. In each set, only one can be marked as default.
+* configuration-reference - A pointer to which `@ConfigurationProperties` are used for that variant.
```

Comment:
Could you add here that there is no limit to how many properties are associated with a variant? That would be my next question when reading this - what are the limitations, etc.